src/Config.cmake.in

```diff
@@ -27,8 +27,11 @@ endif("${isSystemDir}" STREQUAL "-1")
 
 # --------------------------------------------------------------------------------------------------
 # Configure components
-@PACKAGE_INIT@
 
+# Load Radium cmake functions
+include(${CMAKE_CURRENT_LIST_DIR}/RadiumSetupFunctions.cmake)
```

Comment:
This needs to be verified here. What's happen if this was already included ? I remember I had such a case when searching for the Radium package from a subdirectory of the buildtree (e.g making find package(Radium ...) in a CMakeFile.txt in subdir of thee radium source tree) or when several components are searching for Radium. I will not be available to test this next week. I'll do that later if the PR is still opened.